client/web/src/insights/pages/dashboards/DasbhoardsRoutes.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Route, RouteComponentProps, Switch, useRouteMatch } from 'react-router'
+
+import { ExtensionsControllerProps } from '@sourcegraph/shared/src/extensions/controller'
+import { PlatformContextProps } from '@sourcegraph/shared/src/platform/context'
+import { SettingsCascadeProps } from '@sourcegraph/shared/src/settings/settings'
+import { TelemetryProps } from '@sourcegraph/shared/src/telemetry/telemetryService'
+
+import { AuthenticatedUser } from '../../../auth'
+import { Settings } from '../../../schema/settings.schema'
+
+import { InsightsDashboardCreationPage } from './creation/InsightsDashboardCreationPage'
+import { DashboardsPage } from './dashboard-page/DashboardsPage'
+
+export interface DashboardsRoutesProps
+    extends TelemetryProps,
+        SettingsCascadeProps<Settings>,
+        ExtensionsControllerProps,
+        PlatformContextProps<'updateSettings'> {
+    authenticatedUser: AuthenticatedUser
+}
+
+/**
+ * Displays Code Insights dashboard area.
+ */
+export const DashboardsRoutes: React.FunctionComponent<DashboardsRoutesProps> = props => {
+    const { authenticatedUser, settingsCascade, platformContext, telemetryService, extensionsController } = props
+    const match = useRouteMatch()
+
+    return (
+        <Switch>
+            <Route
+                path={`${match.url}/dashboard/:dashboardId?`}
```

Comment:
We usually follow the pattern of naming the _collection_ in plural, e.g. `/users/felixfbecker` and here `/dashboards/:id`
```suggestion
                path={`${match.url}/dashboards/:dashboardId?`}
```